server/tso/allocator_manager.go

```diff
@@ -901,9 +905,11 @@ func (am *AllocatorManager) deleteAllocatorGroup(dcLocation string) {
 	am.mu.Lock()
 	defer am.mu.Unlock()
 	if allocatorGroup, exist := am.mu.allocatorGroups[dcLocation]; exist {
+		log.Info("delete a tso allocator group", zap.String("dc-location", dcLocation), zap.String("member", am.member.Member().Name))
 		allocatorGroup.allocator.Reset()
 		allocatorGroup.leadership.Reset()
 		delete(am.mu.allocatorGroups, dcLocation)
+		allocatorGroup.cancel()
```

Comment:
I think it's better to move it to line 911. Deleting from the map should be the last move to call.

Reply:
I will move the code, but any reason ? I think there is no different